### DISCUSSION TITLE
Icon request: icon-billiards

### DISCUSSION THREAD

**Discussion by heltok** [FortAwesome/Font-Awesome#21213] [Icon Requests]
Hi!
There are a lot of great sports icons and as the World Championship of Snooker is taking place right now I found myself wanting an icon for billiards. Maybe something like this:
![icon-billiards](https://user-images.githubusercontent.com/12615798/39601842-9ebb4c16-4f23-11e8-9328-2f915ecdc6c3.jpg)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Is this a duplicate of #10384?

Even if the design is not the same, do they cover the same use case?

**Comment by heltok** [comment]
OK, I would say that they cover the same issue. Even if I had something like this design in my head.

**Comment by tagliala** [comment]
@heltok it is fine for me to have both

**Comment by Crissov** [comment]
Related Unicode emoji #222:

- [🎱](http://emojipedia.org/billiards/) U+1F3B1 Billiards (CLDR: Magic 8 Ball)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
